.vscode/settings.json

```
@@ -2,6 +2,9 @@
   "[mdx]": {
     "editor.formatOnSave": false
   },
+  "[markdown]": {
+    "editor.formatOnSave": false
+  },
```

```suggestion
  "[mdx][markdown]": {
    "editor.formatOnSave": false
  },
```

こうも書けるんですけど，後々別々に設定を書きたくなる可能性もあると思っています．